i18n/en.pot

```diff
@@ -5,5 +5,83 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
-"POT-Creation-Date: 2020-01-13T11:20:15.557Z\n"
-"PO-Revision-Date: 2020-01-13T11:20:15.557Z\n"
+"POT-Creation-Date: 2020-02-04T21:03:58.202Z\n"
+"PO-Revision-Date: 2020-02-04T21:03:58.202Z\n"
+
+msgid "Metadata Sync"
+msgstr ""
+
+msgid "Data Sync"
+msgstr ""
+
+msgid "SMS Gateway Phone number where SMS are sent"
+msgstr ""
+
+msgid "Confirm SMS Gateway Phone number"
+msgstr ""
+
+msgid "Reserved values downloaded per TEI attribute"
+msgstr ""
+
+msgid "Encrypt DB"
+msgstr ""
+
+msgid "Encrypt"
+msgstr ""
+
+msgid "No"
+msgstr ""
+
+msgid "Yes"
+msgstr ""
+
+msgid "SET TO DEFAULT"
+msgstr ""
+
+msgid "Delete"
+msgstr ""
+
+msgid "Cancel"
+msgstr ""
+
+msgid "CANCEL"
+msgstr ""
+
+msgid "ADD/SAVE"
+msgstr ""
+
+msgid "Download"
+msgstr ""
+
+msgid "DB trimming"
+msgstr ""
+
+msgid "Actions"
+msgstr ""
+
+msgid "Test Android Login"
+msgstr ""
+
+msgid " Enter a user to check access to"
+msgstr ""
+
+msgid "RUN TEST"
+msgstr ""
+
+msgid "{{elementPlural}} global settings"
+msgstr ""
+
+msgid ""
+"Applies to all {{elementPlural}} that an Android user has access to, unless "
+"an specific set of values has been configured for a {{elementSingular}} "
```

Comment:
`an specific set` should be `a specific set`, and `configured for a` will probably be a little cleaner as `configured for` (so that it works with element types that begin with either a vowel or a consonant.